### PR TITLE
new Equation added QTechCost which computes technology cost

### PR DESCRIPTION
**Pull request by riazis**

### CODE REVIEW

File: input.gms [hunk 1]
```diff
@@ -80,4 +80,107 @@ $ondelim
 $include "./iUseEneConvSubTech.csv"
 $offdelim
 ;
-iUsfEnergy(allCy,SBS,EF,TEA,YTIME) = iUsfEneConvSubTech(SBS,EF,YTIME);
\ No newline at end of file
+iUsfEnergy(allCy,SBS,EF,TEA,YTIME) = iUsfEneConvSubTech(SBS,EF,YTIME);
+table iElaSub(allCy,DSBS)                           "Elasticities by subsectors"
```
Comment (giannou) [inline input.gms:84]: units?

File: input.gms [hunk 2]
```diff
@@ -80,4 +80,107 @@ $ondelim
...
+$ondelim$include "./iElaSub.csv"
+$offdelim
+;
+table iConsSizeDistHeat(allCy,conSet)                "Consumer sizes for district heating"
```
Comment (giannou) [inline input.gms:88]: units?

File: input.gms [hunk 3]
```diff
@@ -80,4 +80,107 @@ $ondelim
...
+iAnnConsModCons(runCy,DOMSE) = iConsSizeDistHeat(runCy,"modal");
+
+iAnnSmallCons(runCy,NENSE) = 0.2  ;
+iAnnConsLargCons(runCy,NENSE) = 0.9 ;
+* assuming an average utilisation rate of 0.5 for non-energy uses:
+iAnnConsModCons(runCy,NENSE) = 0.487 ;
+
+iAnnSmallCons(runCy,"BU") = 0.2 ;
+iAnnConsLargCons(runCy,"BU") = 1 ;
+iAnnConsModCons(runCy,"BU") = 0.5 ;
```
Comment (giannou) [inline input.gms:158]: this whole section (lines 92-158) needs to be reworked. to proceed, let's keep as it is for now and I will open an issue (assigning myself) to work on this

**Review by giannou — CHANGES_REQUESTED** [review]

File: preloop.gms [hunk 4]
```diff
@@ -1,4 +1,5 @@
 iCGI(allCy,YTIME) = 1;
+iNcon(SBS) =1;
```
[the remark this answers is not in the document]
Comment (riazis) [reply preloop.gms:2]: It was a test. Resolved now

File: input.gms (hunk 1 above)
Comment (riazis) [reply input.gms:84]: Solved

File: input.gms (hunk 2 above)
Comment (riazis) [reply input.gms:88]: Solved

File: equations.gms [hunk 5]
```diff
@@ -143,9 +143,8 @@ QFuePriSubChp(runCy,DSBS,EF,TEA,YTIME)$(TIME(YTIME) $(not TRANSE(DSBS))  $SECTTE
 VFuePriSubChp(runCy,DSBS,EF,TEA,YTIME)
                 =E=   
              (((VFuelPriceSub(runCy,DSBS,EF,YTIME) + (VRenValue(YTIME)/1000)$(not RENEF(EF))+VCosTech(runCy,DSBS,EF,TEA,YTIME)/1000)/iUsfEnergy(runCy,DSBS,EF,TEA,YTIME)- 
-               (0$(not CHP(EF)) + (VFuelPriceSub(runCy,"OI","ELC",YTIME)*iFracElecPriChp(runCy,YTIME)*VElecIndPrices(runCy,YTIME))$CHP(EF))) + 
-              (0.003) + SQRT( SQR(((VFuelPriceSub(runCy,DSBS,EF,YTIME)+VCosTech(runCy,DSBS,EF,TEA,YTIME)/1000)/VCosTech(runCy,DSBS,EF,TEA,YTIME)- 
-              (0$(not CHP(EF)) + (VFuelPriceSub(runCy,"OI","ELC",YTIME)*iFracElecPriChp(runCy,YTIME)*VElecIndPrices(runCy,YTIME))$CHP(EF)))-(0.003)) + SQR(1e-7) ) )/2;
+               (0$(not CHP(EF)) + (VFuelPriceSub(runCy,"OI","ELC",YTIME)*iFracElecPriChp(runCy,YTIME)*VElecIndPrices(runCy,YTIME))$CHP(EF)))  + SQRT( SQR(((VFuelPriceSub(runCy,DSBS,EF,YTIME)+VCosTech(runCy,DSBS,EF,TEA,YTIME)/1000)/VCosTech(runCy,DSBS,EF,TEA,YTIME)- 
+              (0$(not CHP(EF)) + (VFuelPriceSub(runCy,"OI","ELC",YTIME)*iFracElecPriChp(runCy,YTIME)*VElecIndPrices(runCy,YTIME))$CHP(EF)))) + SQR(1e-7) ) )/2;
```
[the remark this answers is not in the document]
Comment (riazis) [reply equations.gms:147]: Fixed

**Review by giannou — APPROVED** [review]
looks good to me, thanks

**Pull request merged**